Library/Formula/openvpn.rb

```diff
@@ -25,12 +35,22 @@ def install
         "security/pam_appl.h", "pam/pam_appl.h"
     end
 
+    opts = []
+    if build.with? "pkcs11"
+        opts << "--enable-pkcs11"
+        ENV.append_path "PKG_CONFIG_PATH", "/usr/local/lib/pkgconfig"
+        ENV.append "PKCS11_HELPER_CFLAGS", `pkg-config --cflags libpkcs11-helper-1`.chomp
+        ENV.append "PKCS11_HELPER_LIBS",   `pkg-config --libs   libpkcs11-helper-1`.chomp
```

Comment:
Same here. Would prefer passing a path/variable directly rather than shelling out to `pkg-config`. I'm surprised `openvpn` can't pick these up automatically using `pkg-config` without being told explicitly?